### DISCUSSION TITLE
How far away is the v2.7.2?

### DISCUSSION THREAD

**Discussion by ledgeri** [keepassxreboot/keepassxc#8597] [Q&A] [answered]
Just courious... (Possibly within a week, within this calendar month, within a month?)
Also i want to do a database rewamp, what i would not like to repeat soon :), or sit on unused functionality for a half year (...again, like after 2.7.0 vs 2.7.1).

I did not followed the previous versions' progress, so i am blind in this regard.

**Comment by phoerious** [comment, marked as answer]
Within a week.

**Comment by ledgeri** [comment]
Well kept pinky promise! Awsome job guys!
